### ISSUE TITLE
Unable to open requests on single/double clicking on the request from a collection from the Collections Tab

### ISSUE THREAD

**Issue by shashvattrip** [a85/POSTMan-Chrome-Extension#7399]
**Describe the bug**
Unable to open requests on single/double clicking on the request from a collection from the Collections Tab. However, if I right-click on the request and open in a new tab, then the request opens up properly.
Unable to open Examples on single/double clicking on the examples dropdown. The drop-down opens properly, however, the examples don't open.

**To Reproduce**
1. Go to Any collection
2. Click on any request
3. Nothing happens

1. Open any request with an example 
2. Click on Examples dropdown from the top right corner
3. Click on a request's example, nothing happens.
There is no way to open up an example.

**Expected behavior**
Single/double-clicking on any request should open up that request. Same goes for examples

**Screenshots**
If applicable, add screenshots to help explain your problem. Please make sure the screenshot does not contain any sensitive information such as API keys or access tokens. 
<img width="1273" alt="Screenshot 2019-09-27 at 8 31 15 AM" src="https://user-images.githubusercontent.com/2312691/65739080-b9cdcd00-e101-11e9-828c-affc85dbcda2.png">
<img width="641" alt="Screenshot 2019-09-27 at 8 30 54 AM" src="https://user-images.githubusercontent.com/2312691/65739084-bd615400-e101-11e9-987b-3eb6ea02937e.png">

**App information (please complete the following information):**
 - App Type Native App
 - Postman Version 7.8.0 (7.8.0)
 - OS: macOS Mojave 10.14.5 (18F132)

**Additional context**
It wasn't working even with the previous macOS version, but it was working about a month back. It suddenly stopped working about 2 weeks back. Don't remember if I had updated Postman around that time.

**Comment by shashvattrip** [comment]
Still waiting on this

**Comment by lekhanairk2006** [comment]
Seeing same issue in  Postman for Windows
Version 7.20.1
win32 6.1.7601 / x64

**Comment by numaanashraf** [comment]
We have released a fix for this in v7.23.

**Comment by shashvattrip** [comment]
Hey the problem i still present in 7.25.2. Anyways, I've bought a new laptop so dont have this problem on the new laptop

**Comment by galileo-pkm** [comment]
This is still present in v7.29.1
Fedora 32, Postman from flatpak.

**Comment by tyrelcraig** [comment]
This was happening to me on Postman 8.06. 
But I was able to fix it. 

On Mac:
Close Postman
Go to Finder (Cmd+Opt+Space) > Go to folder (Shift+Cmd+G) > Paste ~/Library/Application Support/Postman  > hit return > Delete the files.
Reopen Postman and it should work again.

**Comment by tarockx** [comment]
Having the same problem in latest Postman for Windows, any clues how to fix it?
Started happening randomly, it was working perfectly a few minutes ago...

**Comment by tyrelcraig** [comment]
@tarockx Based on a quick search. 

The similar path to my fix for Windows is:

Open File Explorer and paste in the Path field

C:\Users\<user>\AppData\Local\Postman and
C:\Users\<user>\AppData\Roaming\Postman 

and delete the files in both locations.

Hope that helps!

**Comment by tarockx** [comment]
@tyrelcraig Thank you, it worked.
For anyone reading this in the future: be aware that this action will reset all postman data and settings, it's like doing a fresh installation basically. So make sure you have a backup of your collections and requests (or have an online workspace), otherwise you will lose them all.

**Comment by tyrelcraig** [comment]
@tarockx Glad to hear it. Also good to know, I have mine all stored remotely.

**Comment by vgolovin-vi** [comment]
The problem is still here (Version 8.6.2, linux 5.8.0-55-generic / x64).
"Solved" it by deletion of `~/.config/Postman`

**Comment by sergodeeva** [comment]
Having same problem on v8.7.0, macOS Catalina 10.15.7

**Comment by joeldelvalle** [comment]
This problem continue in version 8.8.0 on mac

**Comment by Netmisha** [comment]
This problem continue in version 8.8.0 on Windows

How to fix:

1. Uninstall Postman
2. Delete C:\Users<user>\AppData\Local\Postman
3. Delete C:\Users<user>\AppData\Roaming\Postman
4. Download latest vesrion of Postman and Install it

**Comment by ImFarel** [comment]
This bug still exist in 8.10.0 on Windows 10

**Comment by ChalangRasul** [comment]
@tyrelcraig. This worked, thank you!!

**Comment by JacobDoug** [comment]
Fixed on windows by doing the following:

Go to %appdata%\Roaming\Postman
Open the window file
Delete the content
Restart postman.

I can now open requests/folders with a single click again.

**Comment by raCerIMA** [comment]
For me, the following worked.

Edit->Settings
Turn on "Always open requests in new tab"

**Comment by scott-nordicwi** [comment]
Just commenting that I saw this issue in Postman. Updated to latest version (9.0.3) today. Resolved it by following steps above.

**Comment by bpejac** [comment]
Have this issue on 9.1.3 on windows 10. Cannot find the setting raCerIMA mentioned. JacobDougs way worked

**Comment by ms3kbtest** [comment]
> Have this issue on 9.1.3 on windows 10. Cannot find the setting raCerIMA mentioned. JacobDougs way worked

On File->Settings now

**Comment by Johnnyevans32** [comment]
> /Library/Application Support/Postman



> For me, the following worked.
> 
> Edit->Settings Turn on "Always open requests in new tab"

this fucking worked! thanks chief

**Comment by beb-midigator** [comment]
I started seeing this on Version 9.15.6 after I restored my M1 MacBook Pro from Time Machine.

I do not see any  ~/Library/Application Support/Postman  or  /Library/Application Support/Postman and changing "Always open requests in new tab" didn't help either.

**Comment by beb-midigator** [comment]
Opened a new issue for this:

https://github.com/postmanlabs/postman-app-support/issues/10807

**Comment by osavchuk-clgx** [comment]
> Fixed on windows by doing the following:
> 
> Go to %appdata%\Roaming\Postman Open the window file Delete the content Restart postman.
> 
> I can now open requests/folders with a single click again.

That works for me, thanks!

**Comment by BryanShekletonwv** [comment]
I had the same issue on 9.29.6.
Quick fix was to open a new postman window (**File -> Open New Postman Window**) and close the old one

**Comment by AbdullahWins** [comment]
> I had the same issue on 9.29.6.
> Quick fix was to open a new postman window (**File -> Open New Postman Window**) and close the old one

It worked!
Thanks mate.

**Comment by NGUYENTAMLONG** [comment]
Thank you for helping me <3<3<3

**Comment by 221443** [comment]
I just had this issue on Windows 10 in Postman v10.21.11, Thank you for helpfull comments <3

**Comment by dkkaran1996** [comment]
The Option is available at Settings.
For Me:

File -> Settings -> general -> User Interface -> Always open sidebar item in new tab - Enabled

**Comment by naulkun7** [comment]
> @tarockx Based on a quick search.
> 
> The similar path to my fix for Windows is:
> 
> Open File Explorer and paste in the Path field
> 
> C:\Users<user>\AppData\Local\Postman and C:\Users<user>\AppData\Roaming\Postman
> 
> and delete the files in both locations.
> 
> Hope that helps!

It worked! Thank you man

**Comment by lcurole** [comment]
This is still busted. Double clicking a tab opens the preview only, not a permanent tab. So when you double click another tab the preview switches to the new tab instead of there now being two permanent open tabs. Changing the setting "Always open sidebar item in new tab - Enabled" allows the opening of permanent tabs but it disables the preview function.

How is such basic UI functionality in PostMan constantly regressing?

**Comment by KushaalKashyap** [comment]
if still someone is having issues this will fix it , File -> Setting ->Always open sidebar item in new tab

**Comment by Bfarncombe** [comment]
@KushaalKashyap That worked for me, thanks!
